percy/environment.py

```diff
@@ -50,13 +50,17 @@ def branch(self):
         raw_branch_output = self._raw_branch_output()
         if raw_branch_output:
             return raw_branch_output
-        # Fourth, fallback to 'master'.
-        utils.print_error('[percy] Warning: unknown git repo, setting PERCY_BRANCH to "master".')
-        return 'master'
+        # Fourth, fallback to NONE.
+        utils.print_error('[percy] Warning: unknown git repo, BRANCH not detected.')
```

Comment:
Same thing as in the other PR, lowercase `BRANCH` so users don't confuse it with an environment variable.